controllers/user.js

```diff
@@ -66,31 +66,43 @@ class Users {
     * @param {object} res - The response object.
     * @param {object} next - The response object.
     */
-  static login(req, res, next) {
-    passport.authenticate('local', { session: false }, (
-      err,
-      user
-    ) => {
-      if (err) {
-        return next(err);
-      }
+  static async login(req, res) {
+    const usernameOrEmail = (req.body.email)
+      ? { email: req.body.email.toLowerCase() }
+      : { username: req.body.username.toLowerCase() };
+    const loginUser = await User
+      .findOne({ where: usernameOrEmail });
+    if (loginUser) {
+      if (bcrypt.compareSync(req.body.password.toLowerCase(),
```

Comment:
The password should not be mutated with any string method; `toLowerCase`, `toUpperCase` or to anything at all. It should remain as it was supplied, else there won't be a match when comparing passwords.

Reply:
ok